composer.json

```diff
@@ -13,11 +13,11 @@
     }
   ],
   "require": {
-    "php": "^5.6 || ^7.0",
-    "phpseclib/phpseclib": "^2.0 !=2.0.8"
+    "php": "^7.2",
+    "phpseclib/phpseclib": "^2.0.23"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.0"
+    "phpunit/phpunit": "^6.0"
```

Comment:
Once the package requires `php-7.2` version at least, the PHPUnit version should be `8.0` at least.

Some useful reference are as follows:

https://phpunit.de/announcements/phpunit-8.html

Reply:
> Once the package requires `php-7.2` version at least, the PHPUnit version should be `8.0` at least.
> 
> Some useful reference are as follows:
> 
> https://phpunit.de/announcements/phpunit-8.html

Hello,

Done.

Thank you!